schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id serial PRIMARY KEY,
```

Comment:
[OPTIONAL]
- [ ] I would advice that you make the *id column not null and autoincremented*, see the below snippet as a guide
`id int NOT NULL IDENTITY(1,1)`